controllers/Backend.php

```diff
@@ -1,104 +1,120 @@
+        $myView->render(array('modos' => $modos, 'errors' => $errors));
     }
 
     public function list()
     {
-        $model_article = new Article();
-        $model_user = new User();
-
-        $page="backend/list";
-        $topbar="backend/topbar";
-        require "views/layout.php";
-    }
-
-    public function logout()
-    {
-        $model_user = new User();
+        $manager_article = new ArticleManager();
+        $posts = $manager_article->get_posts();
+        
+        $manager_user = new UserManager();
+        $users = $manager_user->admin();
+        
+        if($users=0){
+        header("Location:/dashboard");
+        }
         
-        $page="backend/logout";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $myView = new View('list', 'backend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function password()
     {
-        $model_user = new User();
-        
-        $page="backend/password";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $manager_user = new UserManager();
+        $errors = $manager_user->password_verification();
+   
+        if($manager_user->hasnt_password() == 0){
+        header("Location:/dashboard");
+    }
+        $myView = new View('password', 'backend');
+        $myView->render(array('errors' => $errors));
     }
 
     public function postback()
     {
-        $model_article = new Article();
-        $model_user = new User();
-        
-        $page="backend/postback";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        if(isset($_GET['id']))
+        {
+        $id = $_GET['id'];
+        $manager_article = new ArticleManager();
+        $posts = $manager_article->get_post();
+        $errors = $manager_article->postback_verification();
+        }
+
+        if($posts == false){
+            header("Location:/page-erreur-administrateur");
+        }
+
+        else{
+            $myView = new View('postback', 'backend');
+            $myView->render(array('post' => $posts, 'errors' => $errors));
+        }
     }
 
     public function login()
     {
-        $model_user = new User();
+        $manager_user = new UserManager();
+        $errors = $manager_user->login_verification();
         
-        $page="backend/login";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        if(isset($_SESSION['admin'])){
+        header("Location:/dashboard");
+    }
+    
+        $myView = new View('login', 'backend');
+        $myView->render(array('errors' => $errors));
     }
     
     public function new()
     {
-        $model_user = new User();
+        $manager_user = new UserManager();
+        $errors = $manager_user->new_verification();
         
-        $page="backend/new";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        if(isset($_SESSION['admin'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_SESSION Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)